src/components/PlantsV2/components/HighestAndLowestMortalityRateCard.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import OverviewItemCard from 'src/components/common/OverviewItemCard';
+import strings from 'src/strings';
+import { Box, Divider, Typography, useTheme } from '@mui/material';
+import { ObservationResults } from 'src/types/Observations';
+import { useAppSelector } from 'src/redux/store';
+import { selectObservationPlantingZone } from 'src/redux/features/observations/observationPlantingZoneSelectors';
+import { useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
+
+type HighestAndLowestMortalityRateCardProps = {
+  observation?: ObservationResults;
+};
+
+export default function TotalMortalityRateCard({ observation }: HighestAndLowestMortalityRateCardProps): JSX.Element {
+  const theme = useTheme();
+  const defaultTimeZone = useDefaultTimeZone();
+
+  let highestMortalityRate = 0;
+  let highestZoneId: number;
+  observation?.plantingZones.forEach((zone) => {
+    if (zone.mortalityRate > highestMortalityRate) {
+      highestMortalityRate = zone.mortalityRate;
+      highestZoneId = zone.plantingZoneId;
+    }
+  });
```

Comment:
This would fail to pick any zone if you had all zones with mortality rate of `0`.

Reply:
But we want to show _something_, correct? Something would be picked and displayed in the case of a tie that was greater than zero, so it seems like we want to display something if the tie is _at_ zero.

Reply:
Oooh, that's why you were saying to use `>=` . Now I gotcha. Will fix it

Reply:
Thanks!